S/StatsBase/Deps.toml

```diff
@@ -16,16 +16,18 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 ["0.2.4"]
 NumericExtensions = "d47e95ee-b316-5a26-aa50-c41fa0b627f1"
 
-["0.20-0.29"]
+["0.20-0.30"]
 Missings = "e1d29d7a-bbdc-5cf2-9ac0-f12de2c33e28"
 
 ["0.23-0.29"]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
This seems wrong; StatsBase doesn't have a direct dependency on Test.

Reply:
Probably a problem with the conversion script that was used to convert REQUIRE? Anyway it doesn't matter a lot since that's only for old releases.